### ISSUE TITLE
Mod wearablebackpacks is missing the required element 'version' and a version.properties file could not be found.

### ISSUE THREAD

**Issue by WillTereika** [copygirl/WearableBackpacks#7]
As my title says.  This mod was working FINE just yesterday 02/05/2017  And when I went to start minecraft, it gave me the message in the subject line "Mod wearablebackpacks is missing the required element 'version' and a version.properties file could not be found."  in the error report.. I tried uninstalling and reinstalling the mod.. but to no avail, it still says its missing a needed file.  Help?  Below is the entire error report in case anyone can maybe figure it out and tell me what is wrong.

```
-- System Details --
Details:
	Minecraft Version: 1.11.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 343717024 bytes (327 MB) / 989855744 bytes (944 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4096m -Xms256m -XX:PermSize=256m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: 
	Loaded coremods (and transformers): 
FMLPlugin (InventoryTweaks-1.62-dev-77.jar)
  invtweaks.forge.asm.ContainerTransformer
ShetiPhian-ASM (shetiphiancore-1.11.0-3.4.2.jar)
  shetiphian.core.asm.ClassTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 378.49' Renderer: 'GeForce GTX 950/PCIe/SSE2'
[14:38:11] [Client thread/INFO] [FML]: MinecraftForge v13.20.0.2227 Initialized
[14:38:11] [Client thread/INFO] [FML]: Replaced 232 ore recipes
[14:38:11] [Client thread/INFO] [FML]: Found 0 mods from the command line. Injecting into mod discoverer
[14:38:11] [Client thread/INFO] [FML]: Searching C:\Users\Shint\Documents\Curse\Minecraft\Instances\TereiCorp\mods for mods
[14:38:13] [Client thread/WARN] [wearablebackpacks]: Mod wearablebackpacks is missing the required element 'version' and a version.properties file could not be found. Falling back to metadata version 2.1.0
[14:38:13] [Client thread/INFO] [FML]: Forge Mod Loader has identified 26 mods to load
[14:38:14] [Client thread/INFO] [FML]: Attempting connection with missing mods [minecraft, mcp, FML, forge, uncraftingtable, adchimneys, advancedsticks, appleskin, biomesoplenty, chiselsandbits, endercompass, environmentaltech, fastleavedecay, forgeendertech, waila, inventorytweaks, jei, journeymap, harvestcraft, shetiphiancore, terraqueous, valkyrielib, variedcommodities, visiblearmorslots, wailaharvestability, wearablebackpacks] at CLIENT
[14:38:14] [Client thread/INFO] [FML]: Attempting connection with missing mods [minecraft, mcp, FML, forge, uncraftingtable, adchimneys, advancedsticks, appleskin, biomesoplenty, chiselsandbits, endercompass, environmentaltech, fastleavedecay, forgeendertech, waila, inventorytweaks, jei, journeymap, harvestcraft, shetiphiancore, terraqueous, valkyrielib, variedcommodities, visiblearmorslots, wailaharvestability, wearablebackpacks] at SERVER
```

**Comment by copygirl** [comment]
This is just a warning meant for the mod developer. I don't quite know what the proper way to provide the version to remove this warning message is. (Probably what it mentions - the `version.properties`.) This, however, would not cause any issues.

I'm keeping up the issue as a reminder for myself.

----

As for your problem, I'll try to help. Not quite sure what to make of this, but try updating (or downgrading) Forge or either of the two mentioned coremods: InventoryTweaks / ShetiPhianCore.

**Comment by copygirl** [comment]
TIL. So the `Attempting connection with missing mods` is more debug information simply to list mods. Without more information I won't be able to help. What exactly isn't working?

**Comment by copygirl** [comment]
Warning message was fixed in 7de8c80, so I'm closing the issue.
